### PR TITLE
10Th Translation - 1108 Words

### PR DESCRIPTION
**Pull request by cherryloua**
Hi everyone,

Please, take some time to review my PR. Thank you!

### CODE REVIEW

File: book/08-customizing-git/sections/config.asc [hunk 1]
```diff
@@ -427,92 +428,100 @@ $ git config --global core.autocrlf false
...
+Ito ay tumitingin sa anim na pangunahing whitespace na mga isyu -- tatlo ay pinagana sa pamamagitan ng default at maaaring patayin, at tatlo ay di-pinagana sa pamamagitan ng default ngunit pwedeng aktibahin.
+
+Ang tatlong nakapatay sa pamamagitang ng default ay ang `blank-at-eol`, kung saan ay humahanap ng mga patlang sa pagtatapos ng linya; `blank-at-eof`, kung saan ay napapansin ang mga linyang blangko sa dulo ng file; at `space-before-tab`, kung saan ay naghahanap ng mga puwang bago ang mga tab sa simula ng linya.
+
+Ang tatlong di-pinagana sa pamamagitan ng default ngunit pwedeng paganahin ay ang `indent-with-non-tab`, kung saan ay naghahanap ng mga linyang nagsisimula sa puwang imbes na sa mga tab (at kinokontrol sa pamamagitan ng `tabwidth` na opsyon); `tab-in-indent`, kung saan ay nagmamasid sa mga tab sa bahagi ng isang linya; at `cr-at-eol`, kung saan ay nagsasabi sa Git na OK ang carriage return sa dulo ng linya.
+
+Maari mong sabihin sa Git kung saan sa mga ito ang gusto mong paganahin sa pamamagitan ng pagtakda sa `core.whitespace` sa mga halagang gusto mo pwedeng on o off, nakahiwalay sa kuwit
```
Comment (fatelrou) [inline book/08-customizing-git/sections/config.asc]: change "pwedeng" to "maaari"

File: book/08-customizing-git/sections/config.asc [hunk 2]
```diff
@@ -427,92 +428,100 @@ $ git config --global core.autocrlf false
...
 
-The three that are turned on by default are `blank-at-eol`, which looks for spaces at the end of a line; `blank-at-eof`, which notices blank lines at the end of a file; and `space-before-tab`, which looks for spaces before tabs at the beginning of a line.
+Maari mong di-paganahin ang isang opsyun sa pamamagitan ng pag-prepend ng `-` sa harapan ng pangalan nito, o gamitin ang default na halaga sa pamamagitan ng pag-iwan nito sa pangkabuoang pagtakda sa string.
 
-The three that are disabled by default but can be turned on are `indent-with-non-tab`, which looks for lines that begin with spaces instead of tabs (and is controlled by the `tabwidth` option); `tab-in-indent`, which watches for tabs in the indentation portion of a line; and `cr-at-eol`, which tells Git that carriage returns at the end of lines are OK.
+Halimbawa, kung gusto mo ang lahat pwera nalang sa `space-before-tab` ay i-set, maaari mong gawin ito (may `trailing-space` sa pagiging short-hand upang masakop ng pareho ang `blank-at-eol` at `blank-at-eof`):
```
Comment (fatelrou) [inline book/08-customizing-git/sections/config.asc]: change "pwera" to "maliban"

**Review by fatelrou — COMMENTED** [review]
Hi @cherryloua 

I have few suggestion that you might like to implement in your translation. Thanks!

**Comment by cherryloua** [comment]
Thank you for your PR review @fatelrou 

Hi @severinolorillajr I just committed the changes suggested from PR review. Thanks!

**Pull request merged**